packages/next-plugin/src/index.ts

```diff
@@ -2,17 +2,12 @@ import { VanillaExtractPlugin } from '@vanilla-extract/webpack-plugin';
 import browserslist from 'browserslist';
 import { lazyPostCSS } from 'next/dist/build/webpack/config/blocks/css';
 import { findPagesDir } from 'next/dist/lib/find-pages-dir';
-import nextMiniCssExtractPluginExports from 'next/dist/build/webpack/plugins/mini-css-extract-plugin';
+import NextMiniCssExtractPlugin from 'next/dist/build/webpack/plugins/mini-css-extract-plugin';
 
 import type webpack from 'webpack';
 import type { NextConfig } from 'next/types';
 import type { WebpackConfigContext } from 'next/dist/server/config-shared';
 
-// Next.js' built-in mini-css-extract-plugin has a very terrible type definition, let's just use any
-const NextMiniCssExtractPlugin: any =
-  // @ts-expect-error -- Next.js' precompilation does add "__esModule: true", but doesn't add an actual default exports
```

Comment:
// Next.js' built-in mini-css-extract-plugin has a very terrible type definition, let's just use any